go/test/endtoend/vtgate/transaction/trxn_test.go

```diff
@@ -174,3 +174,36 @@ func TestTransactionIsolation(t *testing.T) {
 	utils.AssertMatches(t, conn1, "select id, msg from test where id = 1", `[[INT64(1) VARCHAR("foo")]]`)
 	utils.Exec(t, conn1, "rollback")
 }
+
```

Comment:
File naming:
we have a bunch of files like `tx_*`, one that is `txn_*` and now `trxn_*`.
Can we pick one and stick with it? `tx_*` seems to have come first.